tests/lib/cli.js

```diff
@@ -1160,4 +1161,29 @@ describe("cli", () => {
         });
     });
 
+    describe("testing the cloned config", () => {
+        describe("config file and input file", () => {
+            it("should execute without any errors", async () => {
+                const configPath = getFixturePath("config-file", "cloned-config", "eslintConfig.js");
+                const filePath = getFixturePath("config-file", "cloned-config", "index.js");
+                const code = `--config ${configPath} ${filePath}`;
```

Comment:
This variable name is a bit confusing to me, since this isn't code, but CLI arguments.

Reply:
I will change it 👍